### ISSUE TITLE
Editor: Increase and decrease font size.

### ISSUE THREAD

**Issue by FynnMazurkiewicz** [FynnMazurkiewicz/easypie#16]
This should actually be possible, such functionality is provided by PyQode.
For some reason it doesn't work.

**Comment by FynnMazurkiewicz** [comment]
Zooming is broken because of 
```python
def on_state_changed(self, state):
        if state:
            self.editor.mouse_wheel_activated.connect(
                self._on_wheel_event)
            self.editor.key_pressed.connect(self._on_key_pressed)
            self.mnu_zoom = QtWidgets.QMenu("Zoom")
            a = self.mnu_zoom.addAction(QtGui.QIcon.fromTheme(
                "zoom-in"), "Zoom in")
            a.setShortcut("Ctrl++")
            a.triggered.connect(self.editor.zoom_in)
```
in `/pyqode/core/modes/zoom.py`. 

When triggered.emit() happends, it for reason calls a connected slot with an argument `False`. This in turn is then implicitly converted to 0, so that that zoomlevel += increment has no effect.
**This is a bug in pyQode.**

A workaround is to first type any letter into the editor to enforce keyboard focus of the widget, which will then use _on_key_press ed, which doesn't cause this issue. As soon as keyboard focus is left and signals are used again, the issue will reoccur.

**Comment by FynnMazurkiewicz** [comment]
Fixed with b79198a2604a11343722632c31fb2e26154ed239

**Comment by FynnMazurkiewicz** [comment]
Fixed with b79198a2604a11343722632c31fb2e26154ed239
